frontend/src/components/Toast/ToastContainer.tsx

```diff
@@ -1,6 +1,6 @@
 import { useState, useEffect } from "react";
 import { useRecoilState } from "recoil";
-import { toastListState } from "./toastAtom";
+import { toastListState } from "../../stores/stateToastList";
```

Comment:
```suggestion
import { toastListState } from "@/stores/stateToastList";
```
위의 주소처럼 통일하는 것은 어떤가요?

Reply:
수정하는게 좋겠네요! 감사합니다 :)